enterprise/frontend/src/metabase-enterprise/moderation/components/ModerationStatusIcon/ModerationStatusIcon.tsx

```diff
@@ -1,19 +1,19 @@
-import PropTypes from "prop-types";
-
 import { color } from "metabase/lib/colors";
 import { getStatusIcon } from "metabase-enterprise/moderation/service";
 
+import type { IconProps } from "metabase/core/components/Icon";
 import { Icon } from "metabase/core/components/Icon";
 
-ModerationStatusIcon.propTypes = {
-  status: PropTypes.string,
-};
+type ModerationStatusIconProps = {
+  status?: string;
```

Comment:
`status` may actually be null or undefined:

The types that are used with this component are:
* `CollectionItem / moderated_status?: string;`
* `SearchResult / moderated_status: string | null;`